Formula/glibc.rb

```diff
@@ -152,12 +153,19 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add system ld search paths
+    (prefix/"etc"/"ld.so.conf").atomic_write(ld_so_conf)
   end
 
   def post_install
     # Install ld.so symlink.
     ln_sf lib/"ld-linux-x86-64.so.2", HOMEBREW_PREFIX/"lib/ld.so"
 
+    # Rebuild ldconfig cache
+    rm_f prefix/"etc"/"ld.so.cache"
+    system sbin/"ldconfig"
```

Comment:
Adding `HOMEBREW_PREFIX/lib` in `ld.so.conf` and rebuilding `ldconfig` works for me.

```console
$ cat "$(brew --prefix glibc)"/etc/ld.so.conf
include /home/linuxbrew/.linuxbrew/etc/ld.so.conf.d/*.conf
include /home/linuxbrew/.linuxbrew/opt/glibc/etc/ld.so.conf.d/*.conf
/home/linuxbrew/.linuxbrew/lib
include /etc/ld.so.conf

$ ldconfig -v 2>/dev/null | grep -v ^$'\t'
/home/linuxbrew/.linuxbrew/lib: (from /home/linuxbrew/.linuxbrew/Cellar/glibc/2.35/etc/ld.so.conf:3)
/usr/local/cuda-11.7/targets/x86_64-linux/lib: (from /etc/ld.so.conf.d/cuda-11-7.conf:1)
/usr/lib/x86_64-linux-gnu/libfakeroot: (from /etc/ld.so.conf.d/fakeroot-x86_64-linux-gnu.conf:1)
/usr/lib/wsl/lib: (from /etc/ld.so.conf.d/ld.wsl.conf:4)
/usr/local/lib: (from /etc/ld.so.conf.d/libc.conf:2)
/lib/x86_64-linux-gnu: (from /etc/ld.so.conf.d/x86_64-linux-gnu.conf:3)
/lib32: (from /etc/ld.so.conf.d/zz_i386-biarch-compat.conf:2)
/libx32: (from /etc/ld.so.conf.d/zz_x32-biarch-compat.conf:2)
/home/linuxbrew/.linuxbrew/Cellar/glibc/2.35/lib: (from <builtin>:0)
```

```console
$ source venv/bin/activate
$ python3 -c 'from rich import traceback; traceback.install(); import torch' && echo 'exit with code 0'
exit with code 0

$ cd venv/lib/python3.10/site-packages/torch/lib
$ ldd libtorch_global_deps.so             
        linux-vdso.so.1 (0x00007ffcc1348000)
        libpthread.so.0 => /home/linuxbrew/.linuxbrew/lib/libpthread.so.0 (0x00007fecc9815000)
        libm.so.6 => /home/linuxbrew/.linuxbrew/lib/libm.so.6 (0x00007fecc951e000)
        libdl.so.2 => /home/linuxbrew/.linuxbrew/lib/libdl.so.2 (0x00007fecc9810000)
        libcublas.so.10 => /home/PanXuehai/venv/lib/python3.10/site-packages/torch/lib/./libcublas.so.10 (0x00007fecc5200000)
        librt.so.1 => /home/linuxbrew/.linuxbrew/lib/librt.so.1 (0x00007fecc980b000)
        libnvToolsExt-3965bdd0.so.1 => /home/PanXuehai/venv/lib/python3.10/site-packages/torch/lib/./libnvToolsExt-3965bdd0.so.1 (0x00007fecc4e00000)
        libgomp-a34b3233.so.1 => /home/PanXuehai/venv/lib/python3.10/site-packages/torch/lib/./libgomp-a34b3233.so.1 (0x00007fecc4a00000)
        libc.so.6 => /home/linuxbrew/.linuxbrew/lib/libc.so.6 (0x00007fecc47ef000)
        /home/linuxbrew/.linuxbrew/Cellar/glibc/2.35/lib64/ld-linux-x86-64.so.2 (0x00007fecc982a000)
        libcublasLt.so.10 => /home/PanXuehai/venv/lib/python3.10/site-packages/torch/lib/./libcublasLt.so.10 (0x00007fecc2800000)
        libstdc++.so.6 => /home/linuxbrew/.linuxbrew/lib/libstdc++.so.6 (0x00007fecc259a000)
        libgcc_s.so.1 => /home/linuxbrew/.linuxbrew/lib/libgcc_s.so.1 (0x00007fecc9502000)
```